### DISCUSSION TITLE
Not able to find movment and offset from state in useDrag

### DISCUSSION THREAD

**Discussion by Chiratna** [pmndrs/use-gesture#513] [Q&A] [answered]
I started using @use-gesture/react for my project but I am not able to extract movement and offset from state in useDrag. Does anyone have an idea why is it so? I am using typescript in react project.

**Comment by dbismut** [comment]
Did you look at the docs and all the sandboxes?

**Reply by Chiratna** [reply]
Yes actually they are all destructuring it from the state to get the movement and offset but in my case, it is saying that movement is not defined.

**Reply by Chiratna** [reply]
![image](https://user-images.githubusercontent.com/63202408/177491282-d42eeccf-647a-4dc1-9951-190a3c1a89c5.png)
 
This is how it is being used in the doc. But in my case, I am getting error.

**Reply by dbismut** [reply]
Please send a minimal repro sandbox and I'll have a look.

**Comment by dbismut** [comment, marked as answer]
Related to this https://github.com/pmndrs/use-gesture/issues/501, closing.
